engine/swarm/services.md

```diff
@@ -537,11 +559,52 @@ see
 Swarm services allow you to use resource constraints, placement preferences, and
 labels to ensure that your service is deployed to the appropriate swarm nodes.
 
-### Specify service placement preferences (--placement-pref)
+#### Placement constraints
 
-You can set up the service to divide tasks evenly over different categories of
-nodes. One example of where this can be useful is to balance tasks over a set
-of datacenters or availability zones. The example below illustrates this:
+Use placement constraints to control the nodes a service can be assigned to. In
+the following example, the service only runs on nodes with the
+[label](engine/swarm/manage-nodes.md#add-or-remove-label-metadata)
+`pci_compliant` set to `true`. If no appropriately-labelled nodes are available,
+deployment will fail. The `--constraint` flag uses an equality operator
+(`==` or `!=`). It is possible that all services will run on the same node, or
+each node will only run one replica, or that some nodes won't run any replicas.
+
+```bash
+$ docker service create \
+  --name my-nginx \
+  --replicas 5 \
+  --constraint region==east \
+  nginx
+```
+
+You can also use the `constraint` service-level key in a `docker-compose.yml`
+file.
+
+You can specify multiple placement constraints, and they are `AND`ed together.
```

Comment:
Suggestion: 
You can specify multiple placement constraints by conjoining them with `AND`.

Reply:
That's not what I mean. I mean that if you specify multiple ones, you will only deploy onto nodes that meet all of them.